data.yml

```
@@ -578,6 +578,18 @@ people:
   - company: Stitch Fix
     position: Principal Engineer
     start_day: 2016-10-17
+    end_day: 2017-01-31
+  - company: Georgia General Assembly
+    position: House Representative for District 119
+    start_day: 2017-11-??
```

```suggestion
    start_day: 2017-11-27
```

https://ballotpedia.org/Jonathan_Wallace_(Georgia)